src/Features/Core/Portable/Diagnostics/AnalyzerHelper.cs

```diff
@@ -29,7 +29,6 @@ internal static class AnalyzerHelper
 
         // Shared with Compiler
         internal const string AnalyzerExceptionDiagnosticId = "AD0001";
-        internal const string AnalyzerDriverExceptionDiagnosticId = "AD0002";
```

Comment:
this should be left since it is supposed to be in sync with the one defined in compiler side.